_docs/setup/kubernetes/quick-start-gke-dm.md

```diff
@@ -0,0 +1,246 @@
+deploy/reviews-v1       1         1         1            1           3m
+deploy/reviews-v2       1         1         1            1           3m
+deploy/reviews-v3       1         1         1            1           3m
+
+NAME          HOSTS     ADDRESS         PORTS     AGE
+ing/gateway   *         35.202.120.89   80        3m
+```
+
+Note down the IP and Port assigned to BookInfo product page. (in the example above, its ```35.202.120.89:80```.
+
+
+### Access BookInfo Application
+
+Setup environment variable for the external IP address:
+
+```bash
+kubectl get ingress -o wide
+```
+```bash
+export GATEWAY_URL=35.202.120.89
+```
+
+Verify you can access the BookInfo ```http://${GATEWAY_URL}/productpage```:
+
+![BookInfo](img/dm_bookinfo.png)
+
+Now send some traffic to it:
+```bash
+for i in {1..100}; do curl -o /dev/null -s -w "%{http_code}\n" http://${GATEWAY_URL}/productpage; done
+```
+
+### Verify Installed Istio plugins
+
+
+**Grafana**
+
+Setup a tunnel to Grafana
+
+```bash
+kubectl -n istio-system port-forward $(kubectl -n istio-system get pod -l app=grafana -o jsonpath='{.items[0].metadata.name}') 3000:3000 &
+```
+then 
+```
+ http://localhost:3000/dashboard/db/istio-dashboard
+```
+You should see some statistics for the requests  you just sent earlier:
+
+![Grafana](img/dm_grafana.png)
+
+For more details [About the Grafana Add-on](/docs/tasks/telemetry/using-istio-dashboard.html#about-the-grafana-add-on).
+
+
+**Prometheus**
+
+Prometheus will get installed with Grafana.  You can view the istio and appliation metrics through the console:
+
+```bash
+kubectl -n istio-system port-forward $(kubectl -n istio-system get pod -l app=prometheus -o jsonpath='{.items[0].metadata.name}') 9090:9090 &
+```
+
+View the console at:
+
+```
+ http://localhost:9090/graph
+```
+
+![Prometheus](img/dm_prometheus.png)
+
+For more details [About the Prometheus Add-on](/docs/tasks/telemetry/querying-metrics.html#about-the-prometheus-add-on).
+
+**ServiceGraph**
+
+Setup a tunnel to ServiceGraph
+```bash
+kubectl -n istio-system port-forward $(kubectl -n istio-system get pod -l app=servicegraph -o jsonpath='{.items[0].metadata.name}') 8088:8088 &
+```
+You should see the BookInfo service topology at
+
+```
+http://localhost:8088/dotviz
+```
+
+![ServiceGraph](img/dm_servicegraph.png)
+
+For more details, see [About the SeriviceGrpah Add-on](/docs/tasks/telemetry/servicegraph.html#about-the-servicegraph-add-on).
+
+**Tracing**
+
+Setup a tunnel to Zipkin:
+
+```bash
+kubectl port-forward -n istio-system $(kubectl get pod -n istio-system -l app=zipkin -o jsonpath='{.items[0].metadata.name}') 9411:9411 &
+```
+
+You should see the trace statistics sent earlier:
+
+```
+http://localhost:9411
+```
+
+![Zipkin](img/dm_zipkin.png)
+
+
+For more details on tracing see [Understanding what happened](/docs/tasks/telemetry/distributed-tracing.html#understanding-what-happened).
+
+**Additional Guides**
```

Comment:
## What's next?

You can further explore the BookInfo app and Istio functionality by following any of the tutorials in the [Guides](/docs/guides/) section. However, to do this you need to install `istioctl` to interact with Istio. You can either [install](/docs/setup/kubernetes/quick-start.html#installation-steps) it directly on our workstation or within Cloud Shell

Reply:
replaced existing paragraph